plantcv/plantcv/hyperspectral/read_data.py

```diff
@@ -180,6 +177,86 @@ def read_data(filename):
                   "9": np.complex128, "12": np.uint16, "13": np.uint32, "14": np.int64, "15": np.uint64}
     header_dict["datatype"] = dtype_dict[header_dict["datatype"]]
 
+    return header_dict, wavelength_dict
+
+
+def _parse_arcgis(headername):
+    """Parse a header file and create dictionary of relevant metadata
+
+    Keyword arguments:
+    headername      = File path/name of a hyperspectral data file.
+
+    Returns:
+    header_dict     = Dictionary of hdr metadata
+    wavelength_dict = Dictionary of wavelength metadata
+
+    :param headername: str
+    :return header_dict: dict
+    :return wavelength_dict: dict
+
+    """
+    # Initialize dictionary/lists
+    header_dict = {}
+    wavelength_dict = {}
+    bands_list = []
+    keyword_dict = {"LAYOUT": "interleave", "NROWS": "samples", "NCOLS": "lines", "NBANDS": "bands",
```

Comment:
In ENVI samples are the columns, lines are the rows. So I think these are flipped around here